src/Resources/app/administration/src/module/frosh-tools/component/frosh-tools-tab-scheduled/index.js

```diff
@@ -104,6 +104,28 @@ Component.register('frosh-tools-tab-scheduled', {
             this.createdComponent();
         },
 
+        async scheduleTask(item, immediately = false) {
+            this.isLoading = true;
+
+            try {
+                this.createNotificationInfo({
+                    message: this.$tc('frosh-tools.scheduledTaskScheduleStarted', 0, {'name': item.name})
+                })
+                await this.froshToolsService.scheduleScheduledTask(item.id, immediately);
+                this.createNotificationSuccess({
+                    message: this.$tc('frosh-tools.scheduledTaskScheduleSucceed', 0, {'name': item.name})
+                })
+            } catch (e) {
+                this.createNotificationError({
+                    message: this.$tc('frosh-tools.scheduledTaskScheduleFailed', 0, {'name': item.name})
+                })
```

Comment:
```suggestion
                this.createNotificationInfo({
                    message: this.$t('frosh-tools.scheduledTaskScheduleStarted', {'name': item.name})
                })
                await this.froshToolsService.scheduleScheduledTask(item.id, immediately);
                this.createNotificationSuccess({
                    message: this.$t('frosh-tools.scheduledTaskScheduleSucceed', {'name': item.name})
                })
            } catch (e) {
                this.createNotificationError({
                    message: this.$t('frosh-tools.scheduledTaskScheduleFailed', {'name': item.name})
                })
```

You can use `$t` if you do not need pluralization. Then you can ommit the second parameter.

Reply:
This was copied from https://github.com/FriendsOfShopware/FroshTools/blob/main/src/Resources/app/administration/src/module/frosh-tools/component/frosh-tools-tab-scheduled/index.js#L85-L105